### DISCUSSION TITLE
Feature request: Add a public API to determine if a project enabled dependabot security updates

### DISCUSSION THREAD

**Discussion by pnacht** [community/community#57472] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

When I am evaluating a project's security posture, I'd like to know how they manage their dependencies. One of the most common ways of doing so on GitHub is using dependabot.

Looking from outside, it can often be easy to tell if a project has dependabot enabled: just check if the project has a `.github/dependabot.yml` file.

However, this file is only necessary if the project enabled dependabot's scheduled version bumps. A project can also enable dependabot only for security updates (when a dependency has been attacked and an urgent version bump is necessary). For this functionality, the `.github/dependabot.yml` file is not required.

It's still possible to determine if the project has ever enabled dependabot security updates by checking the project's PRs to see if there were ever any by dependabot. However, it's effectively impossible to determine if a project currently has security updates enabled. After all, if the last dependabot PR was two years ago, is it because the project disabled security updates or because the project's dependencies haven't had any vulnerabilities? What if a project has never received a dependabot PR?

Having a public API (REST or graphQL, without needing an admin token) that simply informs whether the project has security updates enabled would be very useful.

---

Disclosure: My name is Pedro and I work with Google and the [Open Source Security Foundation (OpenSSF)](https://www.openssf.org/) to improve the supply-chain security of the open-source ecosystem.

**Comment by mj5921r** [comment]
تمام اطلاعات موبایل همراهم هک شده کیف پول کریپتو برنامها توسط فردی با کامپیوتر شخصی از آلمان یا آمریکا تمام سرمایه منو دارن سرقت میکنم لطفاً کمکم کنین

**Comment by mj5921r** [comment]
خواهش میکنم کمکم کنین من نمیتونم دسترسی پیدا کنم فقط بعضی مواقع میتونم منو هک کردن وتمام دارایی و اطلاعات حسابم وبسزثت بردن این آدرس ایمیل من mjrafeie522@gmail.com

**Comment by chancej715** [comment]
bump
